internal/servers/controller/handlers/options.go

```diff
@@ -41,3 +42,11 @@ func WithLogger(logger hclog.Logger) Option {
 		o.WithLogger = logger
 	}
 }
+
+// WithAnonymousListing provides an option when creating responses to only include those
+// desired for listing to anonymous users.
+func WithAnonymousListing(anonListing bool) Option {
```

Comment:
mind adding a test?

Reply:
Not sure it needs to be exported.

Reply:
It does, because this is in `handlers`, and it's being used in actual handler implementations.

Reply:
Test added.